package.json

```diff
@@ -52,7 +52,8 @@
   "dependencies": {
     "codemirror": "^5.26.0",
     "codemirror-graphql": "^0.6.11",
-    "marked": "0.3.6"
+    "marked": "0.3.6",
+    "react-tabs": "^2.2.1"
```

Comment:
I'm using the [`react-tabs`](https://github.com/reactjs/react-tabs) to handle all tabbable UI stuff.

Reply:
Would the graphiql team support adding this dependency?